rs/tests/BUILD.bazel

```diff
@@ -311,3 +312,5 @@ uvm_config_image(
     remap_paths = {"activate.sh": "activate"},
     tags = ["manual"],  # this target will be built if required as a dependency of another target
 )
+
+mainnet_images_support()
```

Comment:
I wanted to use genrules to do some simple transforms, but I'm only able to do this at "execute" time. Does this look like a reasonable pattern?

Reply:
Yes, I'm doing something very similar [here](https://github.com/dfinity/ic/pull/1468/files#diff-7fc57714ef13c3325ce2a1130202edced92fcccc0c6db34a72f7b57f60d552a3R103). I'm only surprised you don't need to specify a `name` argument. My `bazel test //bazel:buildifier_test` failed if I omitted that.